src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -302,18 +380,34 @@ private File exportGtfs(String namespace, DataSource dataSource, boolean fromEdi
         return tempFile;
     }
 
+    private class ValuePair {
+        private final Object expected;
+        private final Object found;
+        private ValuePair (Object expected, Object found) {
+            this.expected = expected;
+            this.found = found;
+        }
+    }
+
     /**
      * Run through the list of persistence expectations to make sure that the feed was imported properly into the
      * database.
      */
     private void assertThatImportedGtfsMeetsExpectations(
-        String dbConnectionUrl,
+        Connection connection,
         String namespace,
         PersistenceExpectation[] persistenceExpectations
     ) throws SQLException {
+        // Store field mismatches here (to provide assertion statements with more details).
+        Multimap<String, ValuePair> fieldsWithMismatches = ArrayListMultimap.create();
+        // Check that no validators failed during validation.
+        assertThat(
+            "No validators failed during GTFS import.",
```

Comment:
This string should be reworded as it is a failure message to display in the event that the assertion fails.